### PR TITLE
[RI2] proofreading and removal of some HTML tags

### LINKED ISSUE #2603: [TSC] conform with markdown rules (GSMA compatibility)
GSMA leverages pandoc to convert all CNTT chapters to docx.
Even if the results seem globally ok, a couple of rules would avoid a few manual processing steps.
It's worth mentioning that it obscures any git diff and then makes GSMA updates very difficult

A first set of rules we do apply as soon as possible:
- not to mix html and markdown which could be considered as fine just in GitHub but it's incorrect from a markdown pov (see anchors and images).
- to remove all manual title numberings (in markdown we do only leverage #, ## or ###)
- to remove all our TOCs (it should be rather part of pandoc processing)
- to avoid complex tables as much as possible (MD only accepts basic tabs)
- to modify images rather than applying CSS on it

### PR DESCRIPTION
**Pull request by gkunz**
Partially closes #2603

Signed-off-by: Georg Kunz <georg.kunz@ericsson.com>

### CODE REVIEW

File: doc/ref_impl/cntt-ri2/chapters/chapter01.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
-[<< Back](../)
+﻿# 1. Overview
 
-# 1. Overview
-<p align="right"><img src="../figures/bogo_dfp.png" alt="scope" title="Scope" width="35%"/></p>
+![State](../figures/bogo_dfp.png)
```
Comment (CsatariGergely) [inline doc/ref_impl/cntt-ri2/chapters/chapter01.md:3]:
```suggestion
![State](../figures/bogo_dfp.png) <!-- width="35" -->
```
Let's use this notation for now, so we have a chance to scale the images during the rst conversion.

File: doc/ref_impl/cntt-ri2/chapters/chapter03.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
-[<< Back](../)
+﻿# 3. Requirements for Labs
 
-# 3. Requirements for Labs
-<p align="right"><img src="../figures/bogo_lsf.png" alt="scope" title="Scope" width="35%"/></p>
+![State](../figures/bogo_lsf.png)
```
Comment (CsatariGergely) [inline doc/ref_impl/cntt-ri2/chapters/chapter03.md:3]:
```suggestion
![State](../figures/bogo_lsf.png) <!-- width="35" -->
```

File: doc/ref_impl/cntt-ri2/chapters/chapter04.md [hunk 3]
```diff
@@ -1,7 +1,6 @@
-[<< Back](../)
+﻿# 4. Operational Runbook
 
-# 4. Operational Runbook
-<p align="right"><img src="../figures/bogo_sdc.png" alt="scope" title="Scope" width="35%"/></p>
+![State](../figures/bogo_sdc.png)
```
Comment (CsatariGergely) [inline doc/ref_impl/cntt-ri2/chapters/chapter04.md:3]:
```suggestion
![State](../figures/bogo_sdc.png) <!-- width="35" -->
```

File: doc/ref_impl/cntt-ri2/chapters/chapter05.md [hunk 4]
```diff
@@ -1,7 +1,6 @@
-[<< Back](../)
-
 # 5. Gap Analysis and Development
-<p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
+
+![State](../figures/bogo_ifo.png)
```
Comment (CsatariGergely) [inline doc/ref_impl/cntt-ri2/chapters/chapter05.md:3]:
```suggestion
![State](../figures/bogo_ifo.png) <!-- width="35" -->

```

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

**Review by CsatariGergely — APPROVED** [review]

File: doc/ref_impl/cntt-ri2/chapters/chapter01.md [hunk 5]
```diff
@@ -62,10 +61,10 @@ The main communities involved in driving requirements and development of this RI
...
 
 - **Anuket Functest**
-  - The **Functest** project integrates available test tools and test cases from across the open source ecosystem into a single framework. In particular, it allows to run the test cases selected by the RC2 workstream and outputs a test result package for submission to the OVP program.
+  - The **Functest** project integrates available test tools and test cases from across the open source ecosystem into a single framework. In particular, it allows to run the test cases selected by the RC2 workstream and outputs a test result package for submission to the Anuket Assured program.
 
-- **Compliance and Verification Committee & Cloud Native OVP aka OVP 2.0**
-  - The Compliance and Verification Committee governs the Cloud Native OVP program which owns the definition of the end-to-end framework (tooling, process, tests, review, badging) used to verify conformance of a cloud native infrastructure implementation with the RA2 specifications.
+- **Compliance and Verification Committee & Anuket Assured**
```
Comment (ildikov) [inline doc/ref_impl/cntt-ri2/chapters/chapter01.md:66]: Anuket Assured Program?

File: doc/ref_impl/cntt-ri2/chapters/chapter03.md [hunk 6]
```diff
@@ -1,7 +1,6 @@
-[<< Back](../)
+﻿# 3. Requirements for Labs
```
Comment (ildikov) [inline doc/ref_impl/cntt-ri2/chapters/chapter03.md:1]: Lab Requirements

**Review by ildikov — COMMENTED** [review]
A few nits/questions in line, otherwise looks good.

Thanks,
Ildikó

File: doc/ref_impl/cntt-ri2/chapters/chapter01.md [hunk 7]
```diff
@@ -26,12 +25,12 @@ The RI will be used as follows:
 * To develop the Reference Conformance (RC) document, which will document the tests that can be run against the RI or any Vendor Implementation (VI) to check conformance with the Kubernetes RA.
 * By software vendors to be able to install Kubernetes platforms in their own environments in order to develop against the RI
 
-In order to promote the quick deployment of the RI and avoid duplicating effort across communities, the RI community works closely with CNCF TUG, OVP, ONAP and TIP communities.
+In order to promote the quick deployment of the RI and avoid duplicating effort across communities, the RI community works closely with CNCF TUG, Anuket Assured, ONAP and TIP communities.
```
[the remark this answers is not in the document]
Comment (gkunz) [reply doc/ref_impl/cntt-ri2/chapters/chapter01.md]: included

File: doc/ref_impl/cntt-ri2/chapters/chapter01.md (hunk 5 above)
Comment (gkunz) [reply doc/ref_impl/cntt-ri2/chapters/chapter01.md]: included

File: doc/ref_impl/cntt-ri2/chapters/chapter03.md (hunk 6 above)
Comment (gkunz) [reply doc/ref_impl/cntt-ri2/chapters/chapter03.md]: fixed

**Comment by gkunz** [comment]
Kind review reminder. Thanks.

**Review by rihabbanday — APPROVED** [review]

**Review by pgoyal01 — APPROVED** [review]

**Pull request merged**